README.md

```diff
@@ -138,6 +138,10 @@ The Soulbound is a traitor role who is created by the Soulmage. The Soulbound ca
 **ConVars**
 ```cpp
 ttt_soulbound_max_abilities             4   // The maximum number of abilities the Soulbound can buy. (Set to 0 to disable abilities)
+ttt_soulbound_beebarrel_bees            3   // The number of bees that spawn per exploded beebarrel
+ttt_soulbound_beebarrel_enabled         1   // Whether the place bee barrel ability is enabled or not
+ttt_soulbound_beebarrel_uses            3   // How many uses should of the place bee barrel ability should the Soulbound get. (Set to 0 for unlimited uses)
+ttt_soulbound_beebarrel_cooldown        0   // How long should the Soulbound have to wait between uses of the place bee barrel ability
```

Comment:
Reorder these to match the 'enabled, uses, cooldown, then other' formatting of other abilities. They also need to be moved down the the bottom of the list, just above the Pack-a-Punch ConVars because I have been putting settings that require another mod at the bottom.
```suggestion
ttt_soulbound_beebarrel_enabled         1   // Whether the place bee barrel ability is enabled or not. Requires BeeNade mod
ttt_soulbound_beebarrel_uses            3   // How many uses should of the place bee barrel ability should the Soulbound get. Requires BeeNade mod. (Set to 0 for unlimited uses)
ttt_soulbound_beebarrel_cooldown        0   // How long should the Soulbound have to wait between uses of the place bee barrel ability. Requires BeeNade mod
ttt_soulbound_beebarrel_bees            3   // The number of bees that spawn per exploded beebarrel. Requires BeeNade mod
```

Reply:
done